packages/sku/config/typescript/tsconfig.js

```diff
@@ -24,14 +24,25 @@ module.exports = () => {
       // Fixes https://github.com/cypress-io/cypress/issues/1087
       skipLibCheck: true,
 
-      esModuleInterop: true,
+      // When dependencies have `type: module` TypeScript doesn't know how to resolve them,
+      // so we need to set this to `bundler` or `node`
+      moduleResolution: 'bundler',
+      // This is required by `bundler` and for dynamic `import()`
+      module: 'es2022',
```

Comment:
Confirmed this is still valid with [TypeScript 5.2](https://devblogs.microsoft.com/typescript/announcing-typescript-5-2-beta/#module-and-moduleresolution-must-match-under-recent-node-js-settings)